doc/sphinx/matlab/input-tutorial.rst

```diff
@@ -10,9 +10,9 @@ CTI files
 
 This is the typical way to create a Cantera "phase" object in Matlab::
 
-    gas1 = importPhase('gri30.cti', 'gri30');
+    gas1 = Solution('gri30.cti', 'gri30');
 
-Function ``importPhase`` constructs an object representing a phase of matter by
+Function ``Solution`` constructs an object representing a phase of matter by
```

Comment:
I think the phrasing here needs to change -- `Solution` isn't a function, but the constructor for a class.